### PR TITLE
Issue #5897: verified all modules are in no exception configs

### PR DESCRIPTION
**Pull request by rnveach**
Issue #5897

CI is expected to fail until https://github.com/checkstyle/contribution/pull/348 is merged .

### CODE REVIEW

**Comment by rnveach** [comment]
Expected CI failure looks correct.
````
--- web.txt	2019-03-10 16:57:32.553532073 +0000
+++ file.txt	2019-03-10 16:57:32.561532216 +0000
@@ -1,6 +1,7 @@
 AbbreviationAsWordInName
 AbstractClassName
 AnnotationLocation
+AnnotationOnSameLine
 AnnotationUseStyle
 AnonInnerLength
 ArrayTrailingComma
@@ -16,6 +17,7 @@
 Checker
 ClassDataAbstractionCoupling
 ClassFanOutComplexity
+ClassMemberImpliedModifier
 ClassTypeParameterName
 CommentsIndentation
 ConstantName
@@ -60,9 +62,15 @@
 InterfaceIsType
 InterfaceMemberImpliedModifier
 InterfaceTypeParameterName
+JavadocMethod
+JavadocPackage
 JavadocParagraph
+JavadocStyle
 JavadocTagContinuationIndentation
+JavadocType
+JavadocVariable
 JavaNCSS
+LambdaParameterName
 LeftCurly
 LineLength
 LocalFinalVariableName
@@ -132,6 +140,7 @@
 SuppressWarnings
 SuppressWarningsHolder
 ThrowsCount
+TodoComment
 TrailingComment
 Translation
 TreeWalker
@@ -146,3 +155,4 @@
 VisibilityModifier
 WhitespaceAfter
 WhitespaceAround
+WriteTag
````

File: .ci/travis/travis.sh [hunk 1]
```diff
@@ -387,6 +387,21 @@ check-missing-pitests)
...
+  wget -q \
+    https://raw.githubusercontent.com/checkstyle/contribution/master/checkstyle-tester/checks-nonjavadoc-error.xml
+  wget -q \
+    https://raw.githubusercontent.com/checkstyle/contribution/master/checkstyle-tester/checks-only-javadoc-error.xml
+  xmlstarlet sel --net --template -m .//module -v "@name" \
+    -n checks-nonjavadoc-error.xml -n checks-only-javadoc-error.xml \
+    | grep -vE "Filter" | grep -v "^$" \
+    | sort | uniq | sed "s/Check$//" > web.txt
+  xmlstarlet sel --net --template -m .//module -v "@name" -n config/checkstyle_checks.xml \
+    | grep -vE "Filter|ImportControl" | grep -v "^$" \
```
Comment (rnveach) [inline .ci/travis/travis.sh]: ImportControl and Filters are ignored because it requires an external file.

**Comment by rnveach** [comment]
@romani CI is passing now.

File: .ci/travis/travis.sh (hunk 1 above)
Comment (romani) [reply .ci/travis/travis.sh]: please do 
```
...
MODLUES_WITH_EXTERNAL_FILES="Filter|ImportControl" \
| grep -vE $MODLUES_WITH_EXTERNAL_FILES | grep -v "^$" \
```

to self explain this tiny nuance.

Comment (rnveach) [reply .ci/travis/travis.sh]: Done.

**Comment by rnveach** [comment]
@romani Do you wish to review the changes for the comment.

**Pull request merged**